src/uct/api/uct.h

```diff
@@ -1439,7 +1439,7 @@ struct uct_tag_context {
      * @param [in]  status        Completion status.
      */
      void (*rndv_cb)(uct_tag_context_t *self, uct_tag_t stag, const void *header,
-                     unsigned header_length, ucs_status_t status);
+                     int is_inline, unsigned header_length, ucs_status_t status);
```

Comment:
1) need to update doxygen doc above
2) i'd add this new param in the end, or before the `status`, so that `header_length` goes right after `header`